includes/migration-guide/retargeting/wf/new-ambiguous-dispatcherinvoke-overloads-could-result-different-behavior.md

```diff
@@ -2,7 +2,7 @@
 
 |   |   |
 |---|---|
-|Details|The .NET Framework 4.5 adds new overloads to Dispatcher.Invoke that include a parameter of type @System.Action. When existing code is recompiled, compilers may resolve calls to Dispatcher.Invoke methods that have a @System.Delegate parameter as calls to Dispatcher.Invoke methods with an @System.Action parameter. If a call to a Dispatcher.Invoke overload with a @System.Delegate parameter is resolved as a call to a Dispatcher.Invoke overload with an @System.Action parameter, the following differences in behavior may occur:<ul><li>If an exception occurs, the <xref:System.Windows.Threading.Dispatcher.UnhandledExceptionFilter> and <xref:System.Windows.Threading.Dispatcher.UnhandledException> events are not raised. Instead, exceptions are handled by the <xref:System.Threading.Tasks.TaskScheduler.UnobservedTaskException?displayProperty=name> event.</li><li>Calls to some members, such as <xref:System.Windows.Threading.DispatcherOperation.Result>, block until the operation has completed.</li></ul>|
+|Details|The .NET Framework 4.5 adds new overloads to Dispatcher.Invoke that include a parameter of type <xref:System.Action>. When existing code is recompiled, compilers may resolve calls to Dispatcher.Invoke methods that have a <xref:System.Delegate> parameter as calls to Dispatcher.Invoke methods with an <xref:System.Action> parameter. If a call to a Dispatcher.Invoke overload with a <xref:System.Delegate> parameter is resolved as a call to a Dispatcher.Invoke overload with an <xref:System.Action> parameter, the following differences in behavior may occur:<ul><li>If an exception occurs, the <xref:System.Windows.Threading.Dispatcher.UnhandledExceptionFilter> and <xref:System.Windows.Threading.Dispatcher.UnhandledException> events are not raised. Instead, exceptions are handled by the <xref:System.Threading.Tasks.TaskScheduler.UnobservedTaskException?displayProperty=name> event.</li><li>Calls to some members, such as <xref:System.Windows.Threading.DispatcherOperation.Result>, block until the operation has completed.</li></ul>|
```

Comment:
nit: Dispatcher.Invoke should be a link: \<xref:System.Windows.Threading.Dispatcher.Invoke%2A?displayProperty=nameWithType>
UnobservedTaskException should be displayProperty=nameWithType
DispatcherOperation.Result should also be displayProperty=nameWithType